src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvoker.cs

```diff
@@ -1455,7 +1010,6 @@ private static void Rethrow(ResultExecutedContext context)
         private enum Scope
         {
             Invoker,
```

Comment:
I could call this `Outer`. But it's basically the beginning of the "inner" loop.